string device_type
string device_id
bool enable
---
bool success